lib/modules/manager/github-actions/extract.ts

```diff
@@ -68,6 +70,54 @@ export function extractPackageFile(content: string): PackageFile | null {
       deps.push(dep);
     }
   }
+  return deps;
+}
+
+function extractContainer(container: string | Container): PackageDependency {
+  let dep: PackageDependency;
+  if (typeof container === 'string') {
+    dep = getDep(container);
+  } else {
+    dep = getDep(container?.image);
+  }
+
+  dep.versioning = dockerVersioning.id;
+  return dep;
+}
+
+function extractContainersFromPackageFile(
+  content: string
+): PackageDependency[] {
+  logger.trace('github-actions.extractContainersFromPackageFile()');
+  const deps: PackageDependency[] = [];
+
+  const pkg = load(content, { json: true }) as Workflow;
+
+  Object.entries(pkg.jobs ?? {}).forEach(([, job]: [string, Job]) => {
```

Comment:
use `for...of` loop

Reply:
I tried that, but they do not support typing `[, job]: [string, Job]`. Maybe the issue is also with `js-yaml` which return an `Object` instead of a `Map` as I declared

Reply:
nvm, I figured out the syntax